src/cli/ProjectsCLIController.ts

```diff
@@ -4,32 +4,38 @@ import { BaseCLIController } from './BaseCLIController'
 import { getRepoConfig } from '../utils'
 
 export type Project = {
-  _id: string
-  name: string
+  _id?: string
+  name?: string
   key: string
 }
 
 export class ProjectsCLIController extends BaseCLIController {
+  public async getActiveProject() {
+    const stateProject = StateManager.getFolderState(this.folder.name, KEYS.PROJECT_ID)
+    if (stateProject) {
+      return stateProject
+    }
+    const repoConfig = await getRepoConfig(this.folder)
+    StateManager.setFolderState(this.folder.name, KEYS.PROJECT_ID, repoConfig.project)
+    return repoConfig.project
+  }
+
   public async getAllProjects() {
     const projects = StateManager.getFolderState(this.folder.name, KEYS.PROJECTS)
     if (projects) {
       return projects
     }
-    const { code, error, output } = await this.execDvc('projects get')
+    const { code, error, output } = await this.execDvc('projects list')
```

Comment:
why the change from fetching/storing the full object to only using the key?

Reply:
I'm actually unsure about this. Emma made these changes when she did the initial work. I figured there was a reason for it but I can't seem to find one myself. I'll can change it back and see if anything breaks